src/bindings/js/node/tests/utils.js

```diff
@@ -124,3 +126,138 @@ async function isModelAvailable(model) {
   );
   process.exit(1);
 }
+
+function getAddModel() {
```

Comment:
What is the model? Why can't we download it with the other test models or read it from an XML file?

Reply:
I have put them into .xml files with 1.9K and 1.4K sizes respectively